packages/context-entries/lib/Keys.ts

```diff
@@ -115,6 +117,13 @@ export const KeysInitQuery = {
    * The query's base IRI.
    */
   baseIRI: new ActionContextKey<string>('@comunica/actor-init-query:baseIRI'),
+  /**
+   * Object structure used to cache overloads in sparqlee
+   * (It's here because keeping it across sparqlee evaluators provides a performance increase.)
```

Comment:
I would suggest the following description instead:
"Object to cache function argument overload resolutions. Defaults to an object that is re-created for every query execution."

Reply:
Is it re-created? Shouldn't it be the same cache? That's why we added the setDefault in https://github.com/comunica/comunica/blob/master/packages/actor-init-query/lib/QueryEngineBase.ts#L112 ?

Reply:
In this PR, it's being recreated indeed. If we want to cache it, then we need to store it in the `QueryEngineBase` object, initialize it in the constructor, and set that value in the context in the query method.

Reply:
Sure, let's cache it in the engine!

Reply:
> Object to cache function argument overload resolutions. Defaults to an object that is re-created for every query execution.

Would you remove what is in parentheses?

Reply:
Yes